From 4256ac0daa42107cfaf1ce54dbd1db6ce82975b0 Mon Sep 17 00:00:00 2001
From: Rene Engelhard <rene@debian.org>
Date: Fri, 9 Jun 2017 21:08:31 +0200
Subject: [PATCH] odk settings.mk: add various missing JAVA_PROC_TYPE overrides
 for Linux arches

... which were missing since ~ ever but now break the odk_build-examples test
with /usr/bin/ld: cannot find -ljawt

Change-Id: Ie504ce0c8e21c7d58f1b7e591505940a2f80a2ea
---
 odk/settings/settings.mk | 24 ++++++++++++++++++++++++
 1 file changed, 24 insertions(+)

diff --git a/odk/settings/settings.mk b/odk/settings/settings.mk
index b455a6b5d399..2813959f5dec 100644
--- a/odk/settings/settings.mk
+++ b/odk/settings/settings.mk
@@ -275,6 +275,30 @@ ifeq "$(PROCTYPE)" "powerpc64"
 JAVA_PROC_TYPE=ppc64
 endif
 
+ifeq "$(PROCTYPE)" "powerpc64_le"
+JAVA_PROC_TYPE=ppc64le
+endif
+
+ifeq "$(PROCTYPE)" "arm_eabi"
+JAVA_PROC_TYPE=arm
+endif
+
+ifeq "$(PROCTYPE)" "arm_oabi"
+JAVA_PROC_TYPE=arm
+endif
+
+ifeq "$(PROCTYPE)" "mips_eb"
+JAVA_PROC_TYPE=mips
+endif
+
+ifeq "$(PROCTYPE)" "mips_el"
+JAVA_PROC_TYPE=mipsel
+endif
+
+ifeq "$(PROCTYPE)" "mips64_el"
+JAVA_PROC_TYPE=mips64el
+endif
+
 ifeq "$(PROCTYPE)" "sparc64"
 UNOPKG_PLATFORM=Linux_SPARC64
 JAVA_PROC_TYPE=sparcv9
-- 
2.11.0

